2_base_and_sensor.ipynb

```diff
@@ -436,135 +812,259 @@
+   "metadata": {
+    "lang": "ja"
+   },
    "source": [
     "例えば、`angle_min`, `angle_max`には、レーザスキャナのスキャン範囲（-120度から120度）がラジアンで格納されています。"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "For example, `angle_min` and` angle_max` contain the scan range of the laser scanner (-120 to 120 degrees) in radians:"
+   ]
+  },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
+    "lang": "ja"
+   },
    "outputs": [],
    "source": [
-    "# scan_data.angle_maxの値をラジアンからdegに変換します\n",
+    "# scan_data.angle_maxの値をラジアンからdegに変換します Convert value of scan_data.angle_max from radians to deg\n",
     "scan_data.angle_max / math.pi * 180  # math.pi = π"
    ]
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "ja"
+   },
    "source": [
     "レーザスキャナのデータ本体は、`ranges`という名前の配列に格納されており、配列の長さは721のようです。"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "Actual laser scanner data is stored in the array named `ranges`, and the length of the array seems to be 721:"
+   ]
+  },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
+    "lang": "ja"
+   },
    "outputs": [],
    "source": [
-    "# データの配列の長さを習得\n",
+    "# データの配列の長さを取得 Get length of the array\n",
     "len(scan_data.ranges)"
    ]
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "ja"
+   },
    "source": [
     "スキャン範囲（-120度から120度）上の各観測点がデータ化されているので、配列の真ん中（=361番目）の要素が「ロボット正面から壁までの距離」を表します。\n",
     "\n",
     "センサ値の単位はメートルです。"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "Since each observation point on the scan range (-120 degrees to 120 degrees) is digitized from left to right, the element in the middle of the array (= 361th) represents the \"distance from the front of the robot to the wall\".\n",
+    "\n",
+    "The unit of sensor value is meters."
+   ]
+  },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
+    "lang": "ja"
+   },
    "outputs": [],
    "source": [
-    "# 361番目のデータにアクセス\n",
+    "# 361番目のデータにアクセス Access the 361th data\n",
     "scan_data.ranges[360]"
    ]
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "ja"
+   },
    "source": [
     "# センサ値を使ってロボットを制御する"
    ]
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "# Control the robot using sensor values"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "lang": "ja"
+   },
    "source": [
     "センサ値を使ってロボットを制御してみましょう。"
    ]
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "Let's control the robot using the sensor value."
```

Comment:
"Let's control the robot using sensor values."

I chose this because it does not seem like we have mentioned any specific single sensor to use before this sentence.  So, I went with a more general tone.